prometheus-exporters/elastic-query-exporter/alerts/logs.alerts

```diff
@@ -41,3 +41,16 @@ groups:
         annotations:
           description: "VCSA logs missing in Octobus for vCenter {{ $labels.vcenter }}."
           summary: "VCSA logs missing in Octobus for vCenter {{ $labels.vcenter }}."
+      - alert: VMPoweredDownDuringFailedHostMigration
+        expr: elasticsearch_octobus_vmdown_doc_count
+        for: 5m
+        labels:
+          severity: info
+          tier: vmware
+          service: audit
```

Comment:
Ne service is not `audit`, it is `compute`.
We used audit for the audit related alerts.

Reply:
Fixed